src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let basePrice = days * PRICE_PER_DAY;
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
```

Comment:
The current discount logic applies a fixed discount for rentals longer than 3 days and a different fixed discount for rentals longer than 7 days. However, the task description might require cumulative discounts or a different discount structure. Ensure that the discount logic aligns with the task requirements. If the task specifies cumulative discounts, adjust the logic accordingly.